decidim-elections/app/forms/decidim/votings/check_census_form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # A form to check if data matches census
+    class CheckCensusForm < Form
+      attribute :document_number, String
+      attribute :document_type, String
+      attribute :postal_code, String
+      attribute :day, String
+      attribute :month, String
+      attribute :year, String
+
+      validates :document_number,
+                :document_type,
+                :postal_code,
+                presence: true
+
+      def options_for_document_type_select
+        [
+          I18n.t("passport", scope: "decidim.votings.votings.check_census_form"),
+          I18n.t("dni", scope: "decidim.votings.votings.check_census_form")
+        ]
+      end
+
+      def birthdate
+        year + month + day
```

Comment:
Should we validate this is a correct date? With `Date.new(year, month, day)`

Reply:
💬 I did this in #7777 for the login form, and this will fixed in that PR.